### PR TITLE
God class

### PR DESCRIPTION
**Pull request by ehildebrand**
Make saucer and player not as much of a god class by moving shooting functionality to new classes.

### CODE REVIEW

File: src/main/java/entity/shooters/PlayerShooter.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+/**
+ * Class that regulates shooter activity of Player.
+ * @author Esmee
+ *
+ */
+public class PlayerShooter extends AbstractShooter {
+	private static final float BULLET_SPEED = 4;
+	private static final float BULLET_SIZE = 2;
+    private static final double TRIPLE_SHOT_ANGLE = .1;
```
Comment (SpadesAce) [inline src/main/java/entity/shooters/PlayerShooter.java]: Indentation

File: src/main/java/entity/shooters/PlayerShooter.java [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+import game.Audio;
+
+/**
+ * Class that regulates shooter activity of Player.
+ * @author Esmee
+ *
+ */
+public class PlayerShooter extends AbstractShooter {
+	private static final float BULLET_SPEED = 4;
+	private static final float BULLET_SIZE = 2;
```
Comment (SpadesAce) [inline src/main/java/entity/shooters/PlayerShooter.java]: Bulletsize is increased, is this intentional?

Comment (ehildebrand) [inline src/main/java/entity/shooters/PlayerShooter.java]: nope, i shall fix it at once

**Comment by SpadesAce** [comment]
I say that it is good

**Pull request merged**